src/views/course-edit-tools/course-feedback-download.vue

```diff
@@ -0,0 +1,201 @@
+<!--
+Filename: course-feedback.vue
+Use: download received feedback
+Author: nv
+Date: September 08, 2022
+Dependencies:
+  vuex,
+  @/mixins/locale.vue
+-->
+
+<template>
+  <div class="row mt-5">
+    <div class="col">
+      <b-button
+        size="sm"
+        variant="success"
+        :class="langIsAr? 'float-left' : 'float-right'"
+        @click="$bvModal.show('download-feedback')"
+      >
+        <i class="fas fa-info-circle"></i> {{ y18n('feedback.downloadFeedback') }}
+      </b-button>
+    </div>
+    <div class="col text-dark">
+      {{ y18n('feedback.feedbackTip') }}
+    </div>
+    <b-modal
+      id="download-feedback"
+      :title="y18n('feedback.modal.title')"
+      header-bg-variant="warning"
+      ok-variant="warning"
+      :ok-title="y18n('feedback.modal.ok')"
+      :cancel-title="y18n('cancel')"
+      static
+      centered
+      :aria-label="y18n('popupwarning')"
+      @ok="printPDF"
+    >
+      <p>
+        {{ y18n('feedback.modal.text') }}
+      </p>
+    </b-modal>
+  </div>
+</template>
+
+<script>
+
+import { mapActions, mapGetters } from 'vuex'
+import { storeHandler, locale } from '@/mixins'
+import { jsPDF } from 'jspdf'
+
+export default {
+  name: 'CourseFeedbackDownload',
+
+  mixins: [
+    storeHandler,
+    locale
+  ],
+
+  computed: {
+    ...mapActions([
+      'fetchEnrollmentData'
+    ]),
+    ...mapGetters([
+      'enrollmentFeedback',
+      'course',
+      'courseId'
+    ])
+  },
+
+  methods: {
+    /**
+     * Function getFeedback: gets feedback from store
+     * Author: nv
+     * Last Updated: October 15, 2022 by nv
+     */
+    async getFeedback () {
+      const promise = await this.$store.dispatch('fetchEnrollmentData', { courseId: this.courseId })
+      const feedback = []
+
+      for (const i in promise) {
+        if (promise[i].feedback[promise[i].feedback.length - 1] !== undefined) {
+          feedback[i] = promise[i].feedback[promise[i].feedback.length - 1]
```

Comment:
We did move feedback to an object structure earlier (see https://github.com/LAYAIraq/LAYA-Iraq-Frontend/tree/a47ecb354af0cce482cd55612f331bd310cb4934), why is it reverted to array?